openhtf/util/console_output.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Console output utilities for OpenHTF.
+
+This module provides a few conviences to help formatting output for the CLI, as
+well as for forking output to both a logger (for use with test record loggers)
+and to the CLI directly (through sys.stdout).
+"""
+
+import logging
+import math
+import sys
+import time
+
+import colorama
+import contextlib2 as contextlib
+
+# Colorama module has to be initialized before use.
+colorama.init()
```

Comment:
I'm using colorama in custom output module on Linux and OSX and didn't need to do init

Reply:
The colorama docs specify that it's a no-op on Linux and Mac but not on Windows, so we should just call it to make sure Windows is supported.